src/Mod/Draft/draftmake/make_sketch.py

```diff
@@ -128,49 +176,63 @@ def convertBezier(edge):
         else:
             return(edge)
 
-    rotation = None
-    for obj in objectslist:
+
+    axis = App.Vector(0,0,1).cross(normal)
+    angle = DraftVecUtils.angle(normal, App.Vector(0,0,1)) * App.Units.Radian
+    rotation = App.Rotation(axis, angle)
```

Comment:
I don't think you need to multiply by `Units.Radian` because the result of `angle` is already a value in radians. The `Rotation` class should accept it without issues.

Personally I would define the angle in degrees and only convert it to radians where it is needed. Degrees are just more intuitive if we need to troubleshoot the code later. But if the angle is only used here, then I guess it's fine to keep it short.

Reply:
The angle must be in degrees.